### PR TITLE
17588 - Upgrade to Node 20.5.1 && Fix FAS-UI Library

### PR DESCRIPTION
**Pull request by seeker25**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "@vue/compiler-dom": "^3.2.45",
     "@vue/composition-api": "^1.7.1",
     "axios": "^1.6.0",
-    "fas-ui": "github:bcgov/fas-ui#v1.1.4",
```
Comment (seeker25) [inline auth-web/package.json:28]: Old version was causing a npm error that could only be replicated in CI and Dockerfile

File: auth-web/vite.config.ts [hunk 2]
```diff
@@ -66,7 +66,7 @@ export default defineConfig({
       // Fix for module decorator unit tests fail
       'vuex-module-decorators': path.resolve(__dirname, './node_modules/vuex-module-decorators/dist/esm/index.js'),
       // Fix for multiple vue detected on fas-ui issue, set alias to force fas-ui to use /vue
-      'vue': path.resolve(__dirname, './node_modules/vue/dist/vue.runtime.esm.js')
```
Comment (seeker25) [inline auth-web/vite.config.ts:69]: Component didn't use ESM, reverting back to the regular seemed to fix it

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2710-xrwhxk8f.web.app

**Comment by seeker25** [comment]
Yikes
image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2710-xrwhxk8f.web.app

File: auth-web/vite.config.ts [hunk 3]
```diff
@@ -29,6 +29,10 @@ export default defineConfig({
   define: {
     'import.meta.env.ABOUT_TEXT': generateAboutText(aboutText1, aboutText2)
   },
+  esbuild: {
+    minifySyntax: false,
```
Comment (seeker25) [inline auth-web/vite.config.ts:33]: ` ERROR  [Vue warn]: Invalid component name: "ContractForm$1".` To fix these

**Review by ochiu — APPROVED** [review]

**Comment by seeker25** [comment]
Going to do a release before merging this

**Review by Jxio — APPROVED** [review]

**Pull request merged**